src/lib/array.jl

```diff
@@ -231,6 +230,10 @@ end
   end
 end
 
+@adjoint function sum(xs::AbstractArray{Bool}; dims = :)
```

Comment:
What is about other operations like `prod`?

Reply:
Yes, this is not a general solution this just solves the `sum` case.

Reply:
I see, ok.